awscli/customizations/wizard/ui/prompt.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from prompt_toolkit.application import get_app
+from prompt_toolkit.buffer import Buffer
+from prompt_toolkit.document import Document
+from prompt_toolkit.key_binding import KeyBindings
+from prompt_toolkit.layout.containers import (
+    Window, VSplit, Dimension
+)
+from prompt_toolkit.layout.controls import BufferControl
+
+
+class WizardPrompt:
+    def __init__(self, value_name, value_definition):
+        self._value_name = value_name
+        self._value_definition = value_definition
+        self.container = self._get_container()
+
+    def _get_container(self):
+        answer_container = WizardPromptAnswer(self._value_name)
+        answer_buffer = answer_container.buffer
+        return VSplit(
+            [
+                WizardPromptDescription(
+                    answer_buffer,
```

Comment:
I'm a bit concern about passing the reference to the buffer between containers, maybe it would be safer just to path its name

Reply:
Yeah I'm just going to switch this to use the value name.